src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java

```diff
@@ -51,6 +51,7 @@ private static Cache loadOrGetCache(ModContainer resourceOwner, ResourceType res
 		});
 	}
 
+	// intelliJ says `static` is redundant for records
```

Comment:
remove this comment. We're not doing this.

Reply:
> remove this comment. We're not doing this.

Yea I forgot to take this comment out yesterday.